src/Pages/Auth/Login/index.js

```diff
@@ -21,9 +25,39 @@ function Login(props) {
     history.push("/login")
   }
 
+  const [formData, updateFormData] = useState({ email: "", password: "" })
```

Comment:
Let's follow that convention
```suggestion
  const [formData, setFormData] = useState({ email: "", password: "" })
```